src/xcode/ENA/ENA/Source/HTTPClient/Resources/RegistrationToken/RegistrationTokenResource.swift

```diff
@@ -46,7 +46,8 @@ struct RegistrationTokenResource: Resource {
 	var type: ServiceType
 	var sendResource: PaddingJSONSendResource<RegistrationTokenSendModel>
 	var receiveResource: JSONReceiveResource<RegistrationTokenReceiveModel>
-	
+	var retryingCount: Int?
```

Comment:
Can we have a protocol default implementation that makes adding this to all the resources that don't want to take part in retrying unnecessary?

Reply:
No, we can't because we want to change the property during service usage what is forbidden ("Extensions must not contain stored properties").